proto/osmosis/meshsecurity/v1beta1/tx.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package osmosis.meshsecurity.v1beta1;
+
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos/msg/v1/msg.proto";
+import "gogoproto/gogo.proto";
+import "amino/amino.proto";
+
+option go_package = "github.com/osmosis-labs/mesh-security-sdk/x/meshsecurity/types";
+option (gogoproto.goproto_getters_all) = false;
+
+// Msg defines the wasm Msg service.
```

Comment:
This is supposed to be called by gov, right?

There is no special way of wiring that these days?

Reply:
no, the module would need to provide a CLI command to set and submit this with v1 gov.